WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -408,6 +408,11 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
         }
     }
 
+    override func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
```

Comment:
[react-native-modal](https://github.com/react-native-modal/react-native-modal) (the library Gutenberg's using to display modals) leverages iOS' modal presentation. The method described in [this article](https://medium.com/bumble-tech/intercepting-and-redefining-modal-presentation-from-a-third-party-framework-e91079342443) was the only way I was able to override this, though I'm not sure if this would be considered too hacky.

Reply:
I do not consider this approach to be a hack. Based on my understanding, I think it is valid to hook into `present` to modify the presentation of the modals. 👍🏻